apps/prairielearn/src/pages/instructorAssessmentInstances/instructorAssessmentInstances.ts

```diff
@@ -111,46 +113,55 @@ router.post(
         assessment_instance_id: req.body.assessment_instance_id,
         assessment_id: res.locals.assessment.id,
         time_add: req.body.time_add,
-        time_ref: req.body.time_ref,
         base_time: 'date_limit',
         authn_user_id: res.locals.authz_data.authn_user.user_id,
+        exact_date: formatDate(new Date(), res.locals.course_instance.display_timezone),
```

Comment:
Why not just `new Date()`? `node-postgres` will automatically serialize `Date` instances when they're used in params: https://node-postgres.com/features/types#date--timestamp--timestamptz

If you change this, don't forget to change it below this too.

Reply:
Updated to use `new Date()` in [04498d3](https://github.com/PrairieLearn/PrairieLearn/pull/10800/commits/04498d322330eb9d5e4dfc31e76260a6d9709196). Made this change for `set_time_limit_all` as well.